<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmminorstrata</class>
 <widget class="QWidget" name="frmminorstrata">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="lbHeader">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(247, 247, 247)</string>
     </property>
     <property name="text">
      <string>WRITE SOMETHING HERE LATER &lt;---</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="radioPreexistent">
     <property name="text">
      <string>Choose Pre-existent</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="radioNew">
     <property name="text">
      <string>Create New</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="lineNew"/>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pushDetails">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="app_new.qrc">
       <normaloff>:/app_new/kview.png</normaloff>:/app_new/kview.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QGroupBox" name="groupDetails">
     <property name="title">
      <string>Details</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="4" rowspan="3" colspan="4">
       <widget class="QGroupBox" name="groupActivity">
        <property name="title">
         <string>Activity Status</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" colspan="2">
          <widget class="QRadioButton" name="radioActive">
           <property name="text">
            <string>Active</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radioInactive">
           <property name="text">
            <string>Inactive</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="2" colspan="2">
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Group of Landing Sites</string>
        </property>
        <property name="buddy">
         <cstring>cmbGLS</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="cmbGLS"/>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Start Date</string>
        </property>
        <property name="buddy">
         <cstring>dateTimeEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Comments</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QDateTimeEdit" name="dateTimeEdit">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>End Date</string>
        </property>
        <property name="buddy">
         <cstring>dateTimeEdit_2</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QDateTimeEdit" name="dateTimeEdit_2">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="6">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>379</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QGroupBox" name="groupBackNext">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushPrevious">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Previous</string>
        </property>
        <property name="icon">
         <iconset resource="app_new.qrc">
          <normaloff>:/app_new/back.png</normaloff>:/app_new/back.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushNext">
        <property name="text">
         <string>Next</string>
        </property>
        <property name="icon">
         <iconset resource="app_new.qrc">
          <normaloff>:/app_new/forward.png</normaloff>:/app_new/forward.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="app_new.qrc"/>
 </resources>
 <connections/>
</ui>
